packages/ozone/tests/report-muting.test.ts

```diff
@@ -69,7 +69,7 @@ describe('report-muting', () => {
     })
 
     // Verify that a subject status was not created for bob's post since the reporter was muted
-    await assertSubjectStatus(bobsPostSubject.uri, undefined)
+    await assertSubjectStatus(bobsPostSubject.uri, REVIEWNONE)
```

Comment:
This only needed changing because we are now emitting an additional tag event because of report which causes the status to change to reviewNone which is similar to not having a reviewstate so there is no functional change.